app/org/sagebionetworks/bridge/services/AuthenticationService.java

```diff
@@ -305,7 +305,8 @@ public void verifyEmail(EmailVerification verification) {
         checkNotNull(verification);
 
         Validate.entityThrowingException(EmailVerificationValidator.INSTANCE, verification);
-        accountDao.verifyEmail(verification);
+        Account account = accountWorkflowService.verifyEmail(verification);
+        accountDao.verifyEmail(account);
```

Comment:
It looks like we don't have unit tests for a lot of the methods in AuthenticationService. Should we add them?

Reply:
Between AuthenticationServiceTest and AuthenticationServiceMockTest, all the methods are covered except verifyEmail(...), so I added a couple of tests to the mock test for that.